Class/E-Object-Arrays/exercise3.js

```diff
@@ -20,12 +20,14 @@ let peopleList = [person1, person2, person3];
 // 1. Write a function called describePerson 
 // It should have one input parameter called person
 // It should return a String which says firstName + surname + " is the " + jobTitle
+function describePerson (person){
+    return person.firstName + person.surname + " is the " + person.jobTitle;

```

Comment:
Hey, a little space between `person.firstName` and `person.surname` would be great :) ;)